新增第七章翻译

Caesar-Victory opened the pull request:
专有名词 InnoDB 增译，专有名词 Mysql Shell 保留，比较性部分使用括弧增补，其余部分直译，参考信息中的三级目录保留，不做翻译。

Caesar-Victory committed:
feat: add folder for chapter 7

Caesar-Victory committed:
add translation for C7T

feat: "translation about Chapter 7 Tutorial"

dev2007 commented on docs/7/tutorial.md:
```diff
@@ -0,0 +1,44 @@
+
+**目录**
+
+- [7.1备份和恢复的数据类型](https://dev.mysql.com/doc/refman/8.0/en/backup-types.html)
```
类似这种内部的引用，要调整为内部的路径。比如：(/7/7.1/backup-types.html)

Caesar-Victory replied:
未翻译部分我仅保留中文翻译，URL大部分已经删除，保持内容一致性。

dev2007 commented on docs/7/7.1/backup and recovery type.md:
```diff
@@ -0,0 +1,100 @@
+# 7.1 备份和恢复的数据类型
```
建议翻译完成一篇文档，一小节或者一单后再提交 PR。要不然会混入很多这种英文的进来。即使它没有添加在目录中。

Caesar-Victory replied:
我已收悉，习惯了日拱一卒，每次翻译一部分。但我无法采用递归方式翻译关联部分，因此我决定将章内连接备份后删除，外部链接保留

dev2007 reviewed (commented):
建议PR只提交自己完成的部分。感谢。

Caesar-Victory committed:
modify

improvent: 1. covert English square brackets into Chinese brackets.; 2. remove linked to manual doc URL for principle; 3. add a line wrap; 4. consider that using a Chinese period after the URL may cause browser parsing errors, the translation is consistent with the original text.5. remove wrong text`666666` in 34 line.

Caesar-Victory commented:
请求协助关闭昨日PR，今日我已经修改了URL以及其他内容，推送到自己仓库，发现无法在上次的PR状态下继续推送合并版本，以下为更新内容：

1. 去掉未翻译部分的URL
2. 将方括号从英文替换为中文
3. 增加一行换行
4. 增加约两个空格
5. 因担心URL后跟中文句号会导致浏览器解析错误，遂跟随原文格式
6. 删除第三十四行错误文本“666666”

Sent from Mail for Windows

From: Moon Wu
Sent: Thursday, April 13, 2023 10:19 AM
To: dev2007/mysql8-manual
Cc: Caesar-Victory; Author
Subject: Re: [dev2007/mysql8-manual] 新增第七章翻译 (PR #3)

@dev2007 commented on this pull request.
建议PR只提交自己完成的部分。感谢。

In docs/7/7.1/backup and recovery type.md:
> @@ -0,0 +1,100 @@
+# 7.1 备份和恢复的数据类型
建议翻译完成一篇文档，一小节或者一单后再提交 PR。要不然会混入很多这种英文的进来。即使它没有添加在目录中。
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
